ext/pdo_pgsql/pgsql_driver.c

```diff
@@ -15,6 +15,7 @@
   | Authors: Edin Kadribasic <edink@emini.dk>                            |
   |          Ilia Alshanestsky <ilia@prohost.org>                        |
   |          Wez Furlong <wez@php.net>                                   |
+  |          Pablo Santiago Sánchez <phackwer@gmail.com>                 |
```

Comment:
@phackwer authors mentioned there are usually either original authors or someone who rewrote the significant part of the file. We usually don't put names there just for one fix, but mention them in the NEWS file.

Thanks.